### PR TITLE
Main group 34 reqs 14 e 15

### PR DESCRIPTION
**Pull request by felipedclc**

### CODE REVIEW

**Review by julianadocarmoTI — APPROVED** [review]

File: cypress/integration/header_search_bar_spec.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ describe('14 - Posicione a barra logo abaixo do header e implemente 3 radio butt
   });
 });
 
-describe('15 - Busque na API de comidas caso a pessoa esteja na página de comidas e na de bebidas caso esteja na de bebidas', () => {
+describe.only('15 - Busque na API de comidas caso a pessoa esteja na página de comidas e na de bebidas caso esteja na de bebidas', () => {
```
Comment (henrique3g) [inline cypress/integration/header_search_bar_spec.js:87]:
```suggestion
describe('15 - Busque na API de comidas caso a pessoa esteja na página de comidas e na de bebidas caso esteja na de bebidas', () => {
```

File: src/components/SearchBar.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+export default function SearchBar({ isMealsPage }) {
+  const [inputValue, setInputValue] = useState('');
+  const [radioValue, setRadioValue] = useState('');
+  const [searchItemList, setSearchItemList] = useState([]);
+
+  const handleChange = ({ target }) => {
+    setInputValue(target.value);
+  };
+  console.log(inputValue);
```
Comment (henrique3g) [inline src/components/SearchBar.js:16]:
```suggestion
```

File: src/components/SearchBar.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+  const radioClick = ({ target }) => {
+    setRadioValue(target.value);
+  };
+  console.log(radioValue);
```
Comment (henrique3g) [inline src/components/SearchBar.js:21]:
```suggestion
```

File: src/components/SearchBar.js [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    }
+  };
+
+  const handleClickSearch = () => {
+    if (isMealsPage) return searchMeals();
+    return searchDrinks();
+  };
+
+  console.log(searchItemList);
+
```
Comment (henrique3g) [inline src/components/SearchBar.js:65]:
```suggestion
```

**Review by henrique3g — CHANGES_REQUESTED** [review]
### Excelente! 🚀 #vqv

**Review by henrique3g — APPROVED** [review]

**Pull request merged**